common/persistence/cassandra/cassandraMetadataPersistenceProxy.go

```diff
@@ -74,6 +76,27 @@ func (m *metadataManagerProxy) GetDomain(request *p.GetDomainRequest) (*p.Intern
 	resp, err = m.metadataMgr.GetDomain(request)
 	if err == nil {
 		resp.TableVersion = p.DomainTableVersionV1
+		if !resp.IsGlobalDomain {
```

Comment:
Add a TODO comment to  delete / revert the PR after next next release

Reply:
there is already a TODO at the very top
basically, the proxy file should be deleted once the migration is done